src/externals/mct/configure

```diff
@@ -4311,6 +4311,10 @@ case $ac_fc_v_output in
   *mGLOB_options_string*)
     ac_fc_v_output=`echo $ac_fc_v_output | sed 's/"-mGLOB[^"]*"/ /g'` ;;
 
+  # With Intel ifx, remove -loopopt=0
```

Comment:
Could you please provide some detail re this change.

Reply:
Here is the PR: https://github.com/E3SM-Project/E3SM/pull/3654